INSTALL.md

```diff
@@ -128,15 +128,6 @@ for its options, which include:
         hip             build with HIP/ROCm support
         none            do not build with GPU backend
 
-    gpu_aware_mpi
```

Comment:
Probably not in INSTALL.md, because users could be getting pre-installed versions (e.g., a module on Frontier) and so would never look at INSTALL.md. I will mention it in the CHANGELOG, but we should probably have a section on runtime tuning in https://icl.bitbucket.io/slate/ and discussion in the Users Guide.

Reply:
I added `docs/usage.md` in the Doxygen.